multimedia/libvpx-devel/Portfile

```diff
@@ -159,6 +160,11 @@ post-destroot {
     foreach f [glob ${destroot}${prefix}/lib/${my_name}.*.dylib] {
         system "install_name_tool -id [string map [list ${destroot} ""] ${f}] ${f}"
     }
+    foreach f [glob ${destroot}${prefix}/bin/*] {
+        system "install_name_tool -change \
+        [lindex [split [glob ${destroot}${prefix}/lib/${my_name}.*.dylib] "/"] end] \
+        [string map [list ${destroot} ""] [glob ${destroot}${prefix}/lib/${my_name}.*.dylib]] ${f}"
+    }
 }
```

Comment:
@jmroot / @ryandesign / Anyone, Zhenfu was wondering whether we could potentially improve this. Thoughts...?